content/en/contribute.md

```diff
@@ -24,13 +24,13 @@ If you are new to contributing to open source, we also highly recommend reading
 
 Our community aspires to treat everyone equally and to value all contributions.
 We have a
-[Code of Conduct](http://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html)
+[Code of Conduct](https://docs.scipy.org/doc/scipy/dev/conduct/code_of_conduct.html)
 to foster an open and welcoming environment.
 
 ### Writing code
 
 Programmers, this
-[guide](https://scipy.github.io/devdocs/dev/contributor/development_workflow.html#development-workflow)
+[guide](https://docs.scipy.org/doc/scipy/dev/contributor/development_workflow.html#development-workflow)
```

Comment:
Here too, dev docs is better as we always want to point to latest for such instructions.